index.html

```diff
@@ -31,10 +31,11 @@
                 <p class="connect-link">LET’S CONNECT</p>
             </div>
                 <ul class="connect-icon">
-                    <a href="#" class="fa fa-twitter"></a>
-                    <a href="#" class="fa fa-linkedin"></a>
-                    <a href="#" class="fa fa-angellist"></i></a>
-                    <a href="#" class="fa fa-medium"></i></a>
+                    <li><img src="./image/Linkedinicon.png" alt="" /></li>
+                    <li><img src="./image/github.png" alt="" /></li>
+                    <li><img src="./image/twitter.png" alt="" width="20" height="20" /></li>
+                    <li><img src="./image/M.png" alt="" width="20" height="20" /></li>
+                    <li><img src="./image/fingers.png" alt="" width="20" height="20" /></li>
                 </ul>        

```

Comment:
- [ ] It appears that when making the [previous requested ](https://github.com/benardop/portfolio-project/pull/1#discussion_r912301518)changes, you mistakenly confused the required changes and did the wrong thing. This is what the reviewer meant:

```
<ul class="connect-icon">
<li><a href="#" class="fa fa-twitter"></a></li>
<li><a href="#" class="fa fa-linkedin"></a></li>
<li><a href="#" class="fa fa-angellist"></a></li>
<li><a href="#" class="fa fa-medium"></a></li>
</ul>
```
- Doing this was very correct. You could also change and not use the `<img src="./image/fingers.png" alt="" width="20" height="20" />` as below and it will still be very correct.

```
<ul>
<li><a href="#"><img src="./image/fingers.png" alt="" width="20" height="20" /></a></li>
</ul>
```

**Kindly make sure to always contact the reviewer when you have a required change that you do not fully understand. This will save you a lot of time and frustration**